pcmanfm/application.cpp

```diff
@@ -48,6 +48,7 @@
 #include "applicationadaptor.h"
 #include "preferencesdialog.h"
 #include "desktoppreferencesdialog.h"
+#include <libfm-qt/filepropsdialog.h>
```

Comment:
Please put it beside other `libfm-qt` headers.

Reply:
got it!